backend/application/syson-sysml-import/src/test/resources/ASTTransformerTest/convertNamespaceImportValueTest/model.sysml

```diff
@@ -0,0 +1,4 @@
+package package2 {
+    import package1::part1;
```

Comment:
Could you please specify the visibility of the import?

Reply:
Seems to be outside the scope of the test (for all 3 such request).
I agree that tests on the visibility of import are necessary, especially after the changes of 2.2 but shouldn't they be in their own test?

Reply:
fait.